ting.entities.inc

```diff
@@ -267,32 +267,50 @@ class TingEntity extends DingEntity {
     return !empty($year) ? $year : FALSE;
   }
 
-  public function getSerieNumber() {
-    $serie = $this->splitSerie();
-    // Last part is the number.
-    if (empty($serie[1])) {
-      return FALSE;
-    }
-    return $serie[1];
-  }
-
+  /**
+   * Helper function for processing ting_series.
+   *
+   * @return array|null
+   *   Array of series titles.
+   */
   private function splitSerie() {
-    $serie = $this->get('title_series');
-    if (is_array($serie)) {
-      // Array here does not contain series title and number.
-      // Ex: 100200:23583828.
-      return FALSE;
+    $series = $this->get('title_series');
+    if (!is_array($series)) {
+      $series = array($series);
+    }
+
+    foreach ($series as $k => $serie) {
+      if (strpos($serie, ';')) {
```

Comment:
If ";: will be in beginning of the string strpos will return "0" and if will be skipped